### PR TITLE
Add stickyNote module.

### PR DESCRIPTION
**Pull request by Fjorge0**
The sticky note module is a module that, you guessed it, allows a user to add sticky notes with the `Ctrl+D` keybind. Eventually the plan is to have an icon in the toolbar but this is the best we could do in one day.

Working on this made me 30 seconds late on submitting an assignment, so you better merge it :stuck_out_tongue_winking_eye:.

### CODE REVIEW

File: modules/ui/stickyNotes.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
+let version = '5.6.1';
```
Comment (CanadaHonk) [inline modules/ui/stickyNotes.js]: Version shouldn't start at 5.6.1?

File: modules/ui/stickyNotes.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+function rmMod() {
+    Array.from(document.getElementsByClassName("goosemod-sticky-note")).forEach((note) => note.remove());
+    document.removeEventListener('keypress', newStickyKeybindFunction);
+};
+
+let obj = {
+	// Activating module
+	onImport: async function () {
+		goosemodScope.logger.debug('Sticky Notes', 'Starting...');
+        injMod();
```
Comment (CanadaHonk) [inline modules/ui/stickyNotes.js]: Weird whitespace / spacing here?

**Review by CanadaHonk — CHANGES_REQUESTED** [review]
Going to edit to fix minor changes wanted.

**Review by CanadaHonk — APPROVED** [review]

File: modules/ui/stickyNotes.js (hunk 1 above)
Comment (Fjorge0) [reply modules/ui/stickyNotes.js]: I was counting versions also when it was in development, so the previous version was correct.

**Pull request merged**